go.mod

```diff
@@ -0,0 +1,8 @@
+module dont-slack-evil
```

Comment:
Nice! Peux tu expliquer rapidement à quoi servent `go.mod` et `go.sum` ?

Reply:
`go.mod` keeps track of the dependencies of our module (like `package.json`)
`go.sum` keeps track of each dependency's checksum, to ensure that all downloads are the same amongst the different devs